arelle/ModelDocument.py

```diff
@@ -526,7 +527,7 @@ class Type:
                 "fact dimensions infoset",
                 "html non-XBRL")
 
-    def identify(filesource, filepath):
+    def identify(filesource, filepath) -> int:
```

Comment:
Can we get typehints for the params as well? Here and other places

Reply:
Generally, I haven't touched type hints for params in ancillary files where I'm not sure what they are. I'm expecting we'll get there in due time.